### PR TITLE
governance: Fix PrettyPrint for ChangeParametersProposal

### PR DESCRIPTION
**Pull request by ptrus**
Old:
```
Method: governance.SubmitProposal
Body:
  (invalid)
```

New:
```
Method: governance.SubmitProposal
Body:
  Change Parameters:
    Module: staking
    Changes:
      - Parameter: upgrade_cancel_min_epoch_diff
        Value: 15
```

### CODE REVIEW

File: go/governance/api/api.go [hunk 1]
```diff
@@ -125,12 +126,15 @@ func (p *ProposalContent) Equals(other *ProposalContent) bool {
 // given writer.
 func (p ProposalContent) PrettyPrint(ctx context.Context, prefix string, w io.Writer) {
 	switch {
-	case p.Upgrade != nil && p.CancelUpgrade == nil:
+	case p.Upgrade != nil:
```
Comment (ptrus) [inline go/governance/api/api.go:129]: I think it's fair to assume that the caller validates the proposal before pretty printing it? (that ensures only a single field is set)

**Review by kostko — APPROVED** [review]

File: go/governance/api/api.go (hunk 1 above)
Comment (peternose) [reply go/governance/api/api.go:129]: Or, you could also print all fields if they are set, to not hide that the proposal is invalid.

**Review by peternose — APPROVED** [review]

File: go/governance/api/api.go [hunk 2]
```diff
@@ -233,8 +237,18 @@ func (p *ChangeParametersProposal) Equals(other *ChangeParametersProposal) bool
 // PrettyPrint writes a pretty-printed representation of ChangeParametersProposal to the given
 // writer.
 func (p *ChangeParametersProposal) PrettyPrint(_ context.Context, prefix string, w io.Writer) {
+	var changes map[string]interface{}
+	if err := cbor.Unmarshal(p.Changes, &changes); err != nil {
+		fmt.Fprintf(w, "%s  <error: %s>\n", prefix, err)
```
Comment (peternose) [inline go/governance/api/api.go:242]: Not sure what is better, this
```
Change Parameters:
    <error: cbor: cannot unmarshal positive integer into Go value of type map[string]interface {}>
    <malformed: AQ==>
```
or this 
```
Change Parameters:
    cbor: cannot unmarshal positive integer into Go value of type map[string]interface {}
```

Comment (ptrus) [reply go/governance/api/api.go:242]: No opinions. The `<error>` `<malformed>` is used for unparsable transactions, that's why i used it.

File: go/governance/api/api.go (hunk 1 above)
Comment (ptrus) [reply go/governance/api/api.go:129]: Yeah i guess

**Pull request merged**